### PR TITLE
add clip to overflow-inline

### PR DESCRIPTION
**Pull request by estelle**

### CODE REVIEW

**Comment by dipikabh** [comment]
I'll take a look at this shortly.

File: files/en-us/web/css/overflow-inline/index.md [hunk 1]
```diff
@@ -38,6 +39,8 @@ The `overflow-inline` property is specified as a single keyword chosen from the
   - : Content is clipped if necessary to fit the inline dimension in the padding box. No scrollbars are provided.
 - `scroll`
   - : Content is clipped if necessary to fit in the padding box in the inline dimension. Browsers display scrollbars whether or not any content is actually clipped. (This prevents scrollbars from appearing or disappearing when the content changes.) Printers may still print overflowing content.
+- `clip`
+   - : Overflow content is clipped at the element's overflow clip edge that is defined using the {{CSSXref("overflow-clip-margin")}} property.
```
Comment (yarusome) [inline files/en-us/web/css/overflow-inline/index.md:43]:
```suggestion
  - : Overflow content is clipped at the element's overflow clip edge that is defined using the {{CSSXref("overflow-clip-margin")}} property.
```

File: files/en-us/web/css/overflow-inline/index.md [hunk 2]
```diff
@@ -58,26 +61,32 @@ The `overflow-inline` property is specified as a single keyword chosen from the
...
   </li>
 
   <li>
-    <code>overflow-inline:auto</code> — on most browsers, equivalent to
+    <code>overflow-inline: auto</code> — on most browsers, equivalent to
     <code>scroll</code>
     <div id="div4">ABCDEFGHIJKLMOPQRSTUVWXYZABCDEFGHIJKLMOPQRSTUVWXYZ</div>
   </li>
+  
+  <li>
```
Comment (yarusome) [inline files/en-us/web/css/overflow-inline/index.md:85]:
```suggestion

  <li>
```

File: files/en-us/web/css/overflow-inline/index.md [hunk 3]
```diff
@@ -58,26 +61,32 @@ The `overflow-inline` property is specified as a single keyword chosen from the
 ```html
 <ul>
   <li>
-    <code>overflow-inline:hidden</code> — hides the text outside the box
+    <code>overflow-inline: hidden</code> — hides the text outside the box
```
Comment (dipikabh) [inline files/en-us/web/css/overflow-inline/index.md:64]:
```suggestion
    <code>overflow-inline: hidden</code> (hides the text outside the box)
```

File: files/en-us/web/css/overflow-inline/index.md [hunk 4]
```diff
@@ -58,26 +61,32 @@ The `overflow-inline` property is specified as a single keyword chosen from the
...
     <div id="div1">ABCDEFGHIJKLMOPQRSTUVWXYZABCDEFGHIJKLMOPQRSTUVWXYZ</div>
   </li>
 
   <li>
-    <code>overflow-inline:scroll</code> — always adds a scrollbar
+    <code>overflow-inline: scroll</code> — always adds a scrollbar
```
Comment (dipikabh) [inline files/en-us/web/css/overflow-inline/index.md:69]:
```suggestion
    <code>overflow-inline: scroll</code> (always adds a scrollbar)
```

File: files/en-us/web/css/overflow-inline/index.md [hunk 5]
```diff
@@ -58,26 +61,32 @@ The `overflow-inline` property is specified as a single keyword chosen from the
...
 
   <li>
-    <code>overflow-inline:auto</code> — on most browsers, equivalent to
+    <code>overflow-inline: auto</code> — on most browsers, equivalent to
     <code>scroll</code>
     <div id="div4">ABCDEFGHIJKLMOPQRSTUVWXYZABCDEFGHIJKLMOPQRSTUVWXYZ</div>
   </li>
+
+  <li>
+    <code>overflow-inline: clip</code> — hides the text outside the box beyond the <code>overflow-clip-margin</code>
```
Comment (dipikabh) [inline files/en-us/web/css/overflow-inline/index.md:86]:
```suggestion
    <code>overflow-inline: clip</code> (hides the text outside the box beyond the overflow clip edge)
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks, @estelle. Added some changes to the "See also" section and pointed some nits for other updates.

File: files/en-us/web/css/overflow-inline/index.md [hunk 6]
```diff
@@ -121,6 +135,7 @@ The `overflow-inline` property is specified as a single keyword chosen from the
 
 ## See also
 
-- Related CSS properties: {{cssxref("text-overflow")}}, {{cssxref("white-space")}}, {{Cssxref("overflow")}}, {{Cssxref("overflow-block")}}, {{Cssxref("overflow-x")}}, {{Cssxref("overflow-y")}}, {{Cssxref("clip")}}, {{Cssxref("display")}}
+- [CSS overflow module](/en-US/docs/Web/CSS/CSS_overflow)
+- Related CSS properties: {{cssxref("text-overflow")}}, {{cssxref("white-space")}}, {{Cssxref("overflow")}}, {{Cssxref("overflow-block")}}, {{Cssxref("overflow-x")}}, {{Cssxref("overflow-y")}}, {{CSSXref("overflow-clip-margin")}}, {{Cssxref("clip")}}, {{Cssxref("display")}}
 - [CSS Logical Properties](/en-US/docs/Web/CSS/CSS_Logical_Properties)
 - [Writing Modes](/en-US/docs/Web/CSS/CSS_Writing_Modes)
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/overflow-inline/index.md]: > do we need the link to [logical properties landing page](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties) here?

I don't think so.

**Review by dipikabh — APPROVED** [review]
Thanks 👍

**Pull request merged**